dll/win32/shell32/lang/eu-ES.rc

```diff
@@ -1,7 +1,6 @@
 /*
  * PROJECT:     ReactOS Shell32
  * LICENSE:     LGPL-2.1+ (https://spdx.org/licenses/LGPL-2.1+)
- * PURPOSE:     Spanish (Basque) resource file
```

Comment:
Incorrect deletion actually.

Reply:
No, look at line 7.

Reply:
Well, it says Basque language.

Reply:
Line 7 is for automated tools, it doesn't contain the description of a file.

Reply:
LANGUAGE LANG_BASQUE, SUBLANG_NEUTRAL
contains everything one needs to know, and avoids unnecessary typos like ukraiainian, ukrianian, ukraiaiaian. ukrain.
I asked to please not discuss the headers. You will work over all headers anyway in a later PR.
Either propose anything for further shortening, i would accept such a thing.
Or keep it in your own PR please.